pallets/otc/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = 'pallet-otc'
-version = '1.0.1'
+version = "1.0.2"
```

Comment:
```suggestion
version = '1.0.2'
```

Reply:
Btw. IIRC the convention is to use double quotes in rust Cargo.toml files